cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/styles.scss

```diff
@@ -45,6 +45,7 @@
 
     &:hover {
         transform: scale(1.1);
+        transform-origin: 60% center;
```

Comment:
What do you think if we play with `opacity` instead of transform for hovered menu item?

Like `opacity: 0.6` -> `opacity: 1` on hovered element + transition: 0.5s opacity;

Reply:
Yes, it seems removing the effect would be better for consistency